pkg/compactor/compactor_ring.go

```diff
@@ -24,7 +24,7 @@ type RingConfig struct {
 
 	// Instance details
 	InstanceID             string   `yaml:"instance_id" doc:"hidden"`
-	InstanceInterfaceNames []string `yaml:"instance_interface_names" doc:"hidden"`
+	InstanceInterfaceNames []string `yaml:"instance_interface_names"`
```

Comment:
Since it was hidden until now, should we fix the name to make it consistent with flag?

(Same comment/question applies to other places this PR changes)

Reply:
Good that you spot it! I've renamed the CLI flag to match the YAML config option.

Reply:
You've trained me well :)